starters/cms/src/data.ts

```diff
@@ -38,14 +40,72 @@ export const dataSourceOptions = [
  *   ]
  * }
  */
+
+type ExtendedManagedCollection = ManagedCollection & {
+    dataSourceId: string | null
```

Comment:
Isn't `dataSourceId` the same as the `id` of the collection?

Reply:
`id` is generated by framer-plugin itself meanwhile `dataSourceId` is relative to data.

`const dataSourceId = await collection.getPluginData(PLUGIN_KEYS.DATA_SOURCE_ID)`

Reply:
Should we change the name to something like `externalDatasourceID`?

Reply:
I think that not an issue anymore since i've refactored how to retrieve collection